eureka-client/src/main/java/com/netflix/discovery/endpoint/DnsResolver.java

```diff
@@ -58,15 +53,15 @@ public static DirContext getDirContext() {
      *
      * @return resolved host name
      */
-    public static String resolve(String originalHost) {
+    public String resolve(String originalHost) {
```

Comment:
DnsResolver was written as a utility class with only static methods.  This PR changes an implementation detail of this class so there's no need to make it possible to instantiate DnsResolver directly.  Also, I'm concerned about making such breaking changes to public methods. The PR can be greatly simplified if the static change were to be reverted.

Reply:
@elandau thanks for looking into this.
Yes, i have reverted to static DnsResolver, somehow missed this at first place.